pkg/k8shandler/kibana_handler/clusterkibanarequest.go

```diff
@@ -0,0 +1,82 @@
+package kibana_handler
```

Comment:
i think if we're going to have it in a different package other than `k8shandler` i would prefer it just be `kibana`